src/config-schemas/widget-common-config.ts

```diff
@@ -0,0 +1,5 @@
+import { vitalsConfigSchema } from "./vitals-config-schema";
+
+export const widgetCommonConfigSchema = {
+  vitalsConfig: vitalsConfigSchema
```

Comment:
Wait, what? This isn't common to widgets at all. This is vials-specific. The vitals widget schema shouldn't exist in patient-chart, it should be in patient-chart-widgets, with the vitals widget.

Reply:
I had it in the patient-chart-widget, but you said that I move it over to patient-chart. I can move it over. It's used by multiple widgets and that's why it's in the common configs. 
The approach that you wan't us to use the merging feature will result in duplication in the patient chart config i.e 
```
widgetConfig: [{
name: "vitals-summary",
commonConfig: {
"vitalsConfg": { concepts...}
},
{
name: "vitals-detail",
commonConfig: {
"vitalsConfg": { concepts...}
},
{
name: "vitals-overview",
commonConfig: {
"vitalsConfg": { concepts...}
},
{
name: "vitals-form",
commonConfig: {
"vitalsConfg": { concepts...}
}
]
```

See how the merging solution is repetitive?

Reply:
Would it be easier if you just did this feature or some else? I think we are not reaching an understanding and it's no longer fun to work on this feature. Some of these issues that are being raised came-up earlier on and I thought we reached an understanding?